### DISCUSSION TITLE
hue supported multiple mysql source

### DISCUSSION THREAD

**Discussion by luhea** [cloudera/hue#3414] [Questions] [answered]
Hello，I havae two mysql source want use them in an Hue Server，what should i do？

**Comment by Meepoljdx** [comment, marked as answer]
You can try to config rdbms to achieve this goal, this is my way to connect two PostgreSQL Server:
```
[notebook]
...
  [[interpreters]]
    [[[postgresql2]]]
      name=PostgreSQL2
      interface=rdbms

    [[[postgresql3]]]
      name=PostgreSQL3
      interface=rdbms

[librdbms]
...
  [[databases]]
    [[[postgresql2]]]
      nice_name = "PostgreSQL2"
      name=hue
      engine=postgresql_psycopg2
      host=bigdata-m-002
      port=5432
      user=postgres
      password=XXX
      options={}
      schema=public

    [[[postgresql3]]]
      nice_name = "PostgreSQL3"
      name=metadata
      engine=postgresql_psycopg2
      host=XXXX
      port=5832
      user=postgres
      password=XXXX
      options={}
      schema=public
```
This is the PostgreSQL2
![image](https://github.com/cloudera/hue/assets/63089130/bc785580-43c8-4dd5-944d-4edf99c6d563)
This is the PostgreSQL3
![image](https://github.com/cloudera/hue/assets/63089130/6c52d66e-29c9-4b0d-a7d2-b88f867bdd61)
You can also see this [http://gethue.com/custom-sql-query-editors/](http://gethue.com/custom-sql-query-editors/)

**Reply by luhea** [reply]
Hello，Meepoljdx.  can i config multiple hive？what should i do?

**Reply by Meepoljdx** [reply]
Sorry, I haven't tried, :(. May be you can try jdbc url. But I don't know if hue will be able to support it
